lib/puppet/type/tidy.rb

```diff
@@ -260,8 +260,10 @@ def generate
     end
     found_files = files.find_all { |path| tidy?(path) }.collect { |path| mkfile(path) }
     result = found_files.each { |file| debug "Tidying #{file.ref}" }.sort { |a,b| b[:path] <=> a[:path] }
-    #TRANSLATORS "Tidy" is a program name and should not be translated
-    notice _("Tidying %{count} files") % { count: found_files.size }
+    if found_files.size
```

Comment:
I don't think this does what you want as 0 is truthy:

```ruby
irb(main):007:0> size = 0
=> 0
irb(main):008:0> if size
irb(main):009:1>   puts "truthy"
irb(main):010:1> end
truthy
```

I think you'll want something like `if found_files.size > 0` or `if !found_files.empty?`